### PR TITLE
Clarify drag and drop article handleDrop

### LINKED ISSUE #6051: Didn't call handleDrop at the end
```
function handleDrop(e) {
  e.stopPropagation();

  if (dragSrcEl !== this) {
    dragSrcEl.innerHTML = this.innerHTML;
    this.innerHTML = e.dataTransfer.getData('text/html');
  }

  return false;
}
```

**To Reproduce**
1. Go to bottom of this article
2. find 'handleDrop function'
3. 'As other functions are being used in event listener '
4. handleDrop function isn't being called anywhere and that's why code isn't working

# My code isn't working because I don't know where to call handleDrop function.

### PR DESCRIPTION
**Pull request by samthor**
Fixes #6051

### CODE REVIEW

**Review by rachelandrew — CHANGES_REQUESTED** [review]
This has messed up the code highlighting in the main code section. If you compare the preview with the live article at https://web.dev/drag-and-drop/ I think the highlighted lines need adjusting.

The updated date should also be updated.

**Comment by samthor** [comment]
I reworked this a bit and added a different section to call out adding thie handler.

File: src/site/content/en/blog/drag-and-drop/index.md [hunk 1]
```diff
@@ -205,7 +206,7 @@ consider [throttling or debouncing your event listener](https://css-tricks.com/d
 ## Completing the drop
 
 To process the actual drop,
-add an event listener for the `drop` event.
+add an event listener for the `drop` event (we do this above, so you can fill it in).
```
Comment (rachelandrew) [inline src/site/content/en/blog/drag-and-drop/index.md:209]:
```suggestion
add an event listener for the `drop` event.
```

Comment (rachelandrew) [reply src/site/content/en/blog/drag-and-drop/index.md:209]: now you've changed the structure I think we don't need this.

**Review by rachelandrew — APPROVED** [review]
I think the "we do this above" comment is now redundant, otherwise lgtm.

**Pull request merged**